packages/clay-pagination-bar/src/PaginationBarWithBasicItems.tsx

```diff
@@ -230,6 +233,7 @@ export const ClayPaginationBarWithBasicItems = ({
 					<Picker
 						activeDelta={activeDelta}
 						aria-describedby={ariaDescribedby}
+						aria-label={labels.pagination}
```

Comment:
We need a better name for this variable. This is the aria label for the pagination delta picker. In general our props that are used only as an aria-label, include aria label in the name. If we use this as the title (tooltip) then we could not include ariaLabel in the name. The [date picker](https://github.com/liferay/clay/blob/af20c5531b71b89cd5b95c5805f8e8d8b6915dfb/packages/clay-date-picker/src/DateNavigation.tsx#L17) and [color picker](https://github.com/liferay/clay/blob/af20c5531b71b89cd5b95c5805f8e8d8b6915dfb/packages/clay-color-picker/src/index.tsx#L331) include an `aria-labels` prop that adds all aria labels to that object. So that's another option, but since we only have one aria-label currently that might be too much. So maybe `pageDeltaPickerAriaLabel` or `itemsPerPagePickerTitle` of it makes sense to add this to a title/tooltip then `pageDeltaPickerTitle` or `itemsPerPagePickerTitle`.

Additionally, we need to think about the default label. Maybe `Items Per Page`. This would result in the screen reader reading: "Items Per Page Combo box, 20 items." @marcoscv-work What do you think?

Reply:
We used to use `ariaLabels` to group all of this due to internationalization but last year we changed to using a name that could group `ariaLabels` and other texts, so we are using `messages` now.

I wouldn't add it now because it would introduce a new separate API, but I would keep it as is using `labels` and move it to `messages` in the main version.

Reply:
> 
> Additionally, we need to think about the default label. Maybe `Items Per Page`. This would result in the screen reader reading: "Items Per Page Combo box, 20 items." @marcoscv-work What do you think?

Sounds much better for me "Items Per Page" for the picker, label much more help

Reply:
Thanks @marcoscv-work.

So the variable should probably be called `itemsPerPagePickerAriaLabel` and the default value should be "Items Per Page". Does that sound okay?